test/integration/targets/ansible-galaxy-role/aliases

```diff
@@ -0,0 +1,2 @@
+destructive
```

Comment:
@mattclay what is the criteria for marking the test as destructive?

Reply:
Tests that modify the system, such as installing/uninstalling OS packages, changing global configuration, etc. should be marked as destructive.

Reply:
Removed destructive tag